site/docs/releases.md

```diff
@@ -51,44 +51,60 @@ To add a dependency on Iceberg in Maven, add the following to your `pom.xml`:
-* [\#1536](https://github.com/apache/iceberg/pull/1536) fixes ORC conversion of `notIn` and `notEqual` to match null values
-* [\#1722](https://github.com/apache/iceberg/pull/1722) fixes `Expressions.notNull` returning an `isNull` predicate; API only, method was not used by processing engines
-* [\#1736](https://github.com/apache/iceberg/pull/1736) fixes `IllegalArgumentException` in vectorized Spark reads with negative decimal values
-* [\#1666](https://github.com/apache/iceberg/pull/1666) fixes file lengths returned by the ORC writer, using compressed size rather than uncompressed size
-* [\#1674](https://github.com/apache/iceberg/pull/1674) removes catalog expiration in HiveCatalogs
-* [\#1545](https://github.com/apache/iceberg/pull/1545) automatically refreshes tables in Spark when not caching table instances
+* [\#2091](https://github.com/apache/iceberg/pull/2091) fixes Parquet vectorized reads when column types are promoted
+* [\#1991](https://github.com/apache/iceberg/pull/1991) fixes Avro schema conversions to preserve field docs
+* [\#1981](https://github.com/apache/iceberg/pull/1981) fixes date and timestamp transforms
+* [\#1962](https://github.com/apache/iceberg/pull/1962) fixes Parquet vectorized position reader
+* [\#1811](https://github.com/apache/iceberg/pull/1811) makes refreshing Spark cache optional
+* [\#1798](https://github.com/apache/iceberg/pull/1798) fixes read failure when encountering duplicate entries of data files
+* [\#1785](https://github.com/apache/iceberg/pull/1785) fixes invalidation of metadata tables in CachingCatalog
+* [\#1784](https://github.com/apache/iceberg/pull/1784) fixes resolving of SparkSession table's metadata tables
 
 Other notable changes:
 
-* The `iceberg-hive` module has been renamed to `iceberg-hive-metastore` to avoid confusion
-* Spark 3 is based on 3.0.1 that includes the fix for [SPARK-32168](https://issues.apache.org/jira/browse/SPARK-32168)
-* Hadoop tables will recover from version hint corruption
-* Tables can be configured with a required sort order
-* Data file locations can be customized with a dynamically loaded `LocationProvider`
-* ORC file imports can apply a name mapping for stats
+* NaN counter is added to format v2 metrics
+* Shared catalog properties are added in core library to standardize catalog level configurations
+* Spark and Flink now supports dynamically loading customized `Catalog` and `FileIO` implementations
+* Spark now supports loading tables with file paths via HadoopTables
+* Spark 2 now supports loading tables from other catalogs, like Spark 3
+* Spark 3 now supports catalog names in DataFrameReader when using Iceberg as a format
+* Hive now supports INSERT INTO, case insensitive query, projection pushdown, create DDL with schema and auto type conversion
+* ORC now supports reading tinyint, smallint, char, varchar types
+* Hadoop catalog now supports role-based access of table listing
```

Comment:
This can be removed because it is misleading. Role-based access isn't what was supported. The fix was to avoid failing to list when some tables cannot be read due to permissions errors. Instead of failing, the tables are ignored.